.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,70 @@
+> Please make sure you read this and check all the appropriate sections.
+>
+> Select one of the following:
+>
+> I am filing a bug.
+>
+> > In this case you need the following information:
+> >
+> > (you can just delete this whole template and enter the number followed by
+> > your answer).
+> >
+> > 1. OS. Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
+> > 2. Versions. Please run `ng --version`. If there's nothing outputted, please
+> >    run in a Terminal: 
+> > 
+> >         node --version
+> >         npm --version
+> >         npm info angular-cli
+> >
+> >    And paste the result here.
+> > 3. Repro steps. Tell us exactly what you did and how you came to the failure. 
+> >    If you can give us steps from the moment you created an empty application,
+> >    even better! Please try to be as descriptive as you can; this will avoid
+> >    unnecessary back and forth. Was this an app that wasn't created using the
+> >    CLI? What change did you do on your code? etc.
+> > 4. The log given by the failure. Normally this include a stack trace and some
+> >    more information.
+> > 5. Mention any other details that might be useful.
+> >
+> > Thanks! We'll be in touch soon. Allow a day or two, in case all of us
+> > are out from exhaustion :)
+>
+> ----
+>
+> I am filing a feature request.
+>
+> > Unfortunately, for now, although we accept proposals, it will be probably be
+> > deferred. It's not that we don't want to hear you, it's that we're focusing
+> > on the beta release and as such we want to keep things a bit tidier than it
+> > will be once this is out in the open.
+> >
+> > We're not closed to good ideas, we simply have a limited amount of bandwidth.
+> > There's only 24 hours per day.
+>
+> ----
+>
+> I want customer support / I don't know how to do X.
+>
+> > This is unfortunately not the right place. Please join the discussion on
+> > gitter, Stack Overflow or twitter and don't be afraid to post your
+> > questions there. A FAQ will be created with those and it will help answer
+> > questions other people might have as well.
+> >
+> > Please understand that the Angular CLI is still maturing, and before Beta
+> > there WILL be breaking changes and a lack of support is to be expected.
+> > Trust us; we want this to be ready as fast as you do, which is why we're
+> > busy working really hard to make this tool the best there is to help you
+> > build your awesome web applications!
+>
+> ----
+>
+> I want to help!
+>
+> > Please read our contributing guidelines;
+> > https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md
+>
+> > Do not file an issue. These will be closed without an explanation. If
```

Comment:
I get what you're saying here, but the `closed without an explanation` sounds just a bit hostile.

If people want to help, I think the last bit of this paragraph is enough (from "If (instead) you really want...").